src/nunavut/lang/cpp/__init__.py

```diff
@@ -968,15 +949,80 @@ def filter_destructor_name(language: Language, instance: pydsdl.Any) -> str:
 
 
 @template_language_filter(__name__)
-def filter_default_value_initializer(language: Language, instance: pydsdl.Any) -> str:
+def filter_value_initializer(language: Language, instance: pydsdl.Any, initialization_context: SpecialMethods) -> str:
```

Comment:
@pavel-kirienko, what do you think of my CLI proposal here?

Reply:
lgtm

Reply:
This sounds like a good plan though it will take me some time to sort out the best way to implement it.  Doing a quick search I don't see much logic hinging on the specified language standard -- is there not much use of that currently?  Do you think should try to keep to a single _composite_type.j2 template or break it out into multiples?  Let me know if you have any suggestions on a general strategy otherwise I'll dive in and figure something out.